jwst/cube_build/cube_build_wcs_util.py

```diff
@@ -153,7 +153,7 @@ def find_corners_NIRSPEC(input, instrument_info, coord_system):
     lambda_slice = np.zeros(nslices * 2)
     k = 0
     # for NIRSPEC there are 30 regions
-    log.info('Looping over slices to determine cube size .. this takes a while')
+    log.info('Looping over slices to determine cube size')
```

Comment:
So it's faster now? ;-)

Reply:
Yes it is faster. But that was work that was on the wcs side of things. I think William might have worked on it